lib/EnvironmentPlugin.js

```diff
@@ -1,17 +1,25 @@
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Simen Brekken @simenbrekken
+	Authors Simen Brekken @simenbrekken, Einar Löve @einarlove
 */
 var DefinePlugin = require("./DefinePlugin");
 
 function EnvironmentPlugin(keys) {
-	this.keys = Array.isArray(keys) ? keys : Array.prototype.slice.call(arguments);
+	if (typeof keys === 'string') {
```

Comment:
I'll get on it. Had issues with definePlugin not working as expected in the 2.x.x beta.